contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -6,18 +6,28 @@
                         unicode_literals, with_statement)
 
 from pants.base.payload import Payload
+from pants.base.payload_field import PrimitiveField
 
 from pants.contrib.node.targets.node_package import NodePackage
 
 
 class NodeModule(NodePackage):
   """A Node module."""
 
-  def __init__(self, sources=None, address=None, payload=None, **kwargs):
+  def __init__(
+    self, sources=None, build_script=None, output_dir='dist',
+    preserve_artifacts=True, address=None, payload=None, **kwargs):
     """
     :param sources: Javascript and other source code files that make up this module; paths are
                     relative to the BUILD file's directory.
     :type sources: `globs`, `rglobs` or a list of strings
+
+    :param build_script: build script name as defined in package.json.
+    :param output_dir: relative path to assets generated by build script. The path will be
```

Comment:
A longer example would be good here: this all gets rendered to http://www.pantsbuild.org/build_dictionary.html , so it's worthwhile!